### ISSUE TITLE
Add locale-filter to Search Interface

### ISSUE THREAD

**Issue by rmzelle** [zotero/styles-repo#3]
It would be nice if users could filter styles on the language specified in `default-locale`. I think a dropdown-menu would suffice.

If limited to the first two characters of the locale code (e.g. "en" for "en-US", "en-GB" and "en"), the number of different languages isn't that big. https://github.com/citation-style-language/locales/blob/master/locales.json provides a conversion table between locale codes and English language names.

See also http://xbiblio-devel.2463403.n2.nabble.com/Filter-by-locale-in-Zotero-Style-Repository-td7578322.html

**Comment by rmzelle** [comment]
I still think this is valuable since it's not immediately obvious to users that they have to type in English language names ("German" instead of "Deutsch"). We also currently don't automatically check whether all styles with a `default-locale` attribute mention the locale name in the title, so relying on `default-locale` would be more reliable.

We would just have to make it clear somehow that language-agnostic styles (without a `default-locale`) can also be used for the locale of interest.

**Comment by zuphilip** [comment]
I agree that language would be a useful filter. However, I think that a clear handling of the language-agnostic (or call them language-independent?) styles is important. Maybe, we could have a list of all different languages _AND_ an option to include/exclude language-agnostic styles.
